src/bpmn-visu-js/mx-graph/Graph.ts

```diff
@@ -0,0 +1,23 @@
+import { mxgraph, mxgraphFactory } from './../../../node_modules/ts-mxgraph/index';
```

Comment:
❓ waiting for file name convention

Reply:
we will follow airbnb convention
ok for approval then